Examples/Run/Common/src/CommonOptions.cpp

```diff
@@ -63,6 +63,11 @@ void ActsExamples::Options::addGeometryOptions(
       "Sub detectors for the output writing");
 }
 
+void ActsExamples::Options::addFittingOptions(
```

Comment:
I guess it's better to put the fitting options in a separate file e.g. `TrackFittingOptions.cpp(hpp)` under the `TrackFitting` directory.